pkgs/by-name/ca/cargo-smart-release/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  git,
+  pkg-config,
+  openssl,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "cargo-smart-release";
+  version = "0.21.8";
+
+  src = fetchFromGitHub {
+    owner = "GitoxideLabs";
+    repo = "cargo-smart-release";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-T7rdNncK3nukjQb90A49jlLqs4Gy/HUUs7yiDi76GrU=";
+  };
+
+  cargoHash = "sha256-oFuOf7RtV0nRHk0woV11gdqw0mtFskTydC6L/cZf6JE=";
+
+  nativeBuildInputs = [
+    git
+    pkg-config
+  ];
+
+  buildInputs = [ openssl ];
```

```suggestion
  nativeBuildInputs = [
    pkg-config
    makeBinaryWrapper
  ];

  buildInputs = [ openssl ];
  
  postFixup = ''
    makeWrapper $out/bin/cargo-smart-release --prefix PATH : ${lib.makeBinPath [ git ]}
  '';
```

if tests require git:

```
nativeCheckInputs = [ git ];
```

---

Really? Or both? https://github.com/search?q=repo%3AGitoxideLabs%2Fcargo-smart-release%20git&type=code

Also I don't understand how git would work with a tarball without .git

---

My intention is just to leave the `git` command executed during runtime to user's `git`, but if it is better to wrap the final binary then I will make the change.

Also the current `build.rs`'s `git` command does nothing, I just added it there so that the build process does not fail.